operator/src/main/java/oracle/kubernetes/operator/Watcher.java

```diff
@@ -87,11 +87,15 @@ void start(ThreadFactory factory) {
   }
 
   private void doWatch() {
-    setIsDraining(false);
+    try {
+      setIsDraining(false);
 
-    while (!isDraining()) {
-      if (isStopping()) setIsDraining(true);
-      else watchForEvents();
+      while (!isDraining()) {
+        if (isStopping()) setIsDraining(true);
+        else watchForEvents();
+      }
+    } catch (Exception e) {
+      LOGGER.warning("Exception caught in thread " + Thread.currentThread().getName(), e);
```

Comment:
INFO and above need to be internationalized.

Reply:
Debug code I forgot to remove